### ISSUE TITLE
Performance issues

### ISSUE THREAD

**Issue by cameronnapoli** [ruucm/shadergradient#62]
Your landing page has some performance issues. I'm using Chrome Version 119.0.6045.199

**Comment by ruucm** [comment]
yeah we've used lot of canvas at there. We are refactoring the whole website & package now, and also thinking about using https://github.com/pmndrs/tunnel-rat patterns like [react-three-next](https://github.com/pmndrs/react-three-next) dose.
